packages/m2m-gateway/src/clients/clientsProvider.ts

```diff
@@ -73,6 +74,9 @@ export type AuthorizationProcessClient = {
 export type EServiceTemplateProcessClient = ZodiosClientWithMetadata<
   ReturnType<typeof eserviceTemplateApi.createProcessApiClient>
 >;
+export type EventManagerClient = ZodiosClientWithMetadata<
+  ReturnType<typeof m2mEventApi.createM2mEventsApiClient>
+>;
```

Comment:
Here we could also not need the metadata patch since M2M events are not event-sourced resources, they don't have data + metadata version, etc.

```suggestion
export type EventManagerClient = ReturnType<typeof m2mEventApi.createM2mEventsApiClient>
```